vpr/src/base/read_place.h

```diff
@@ -1,16 +1,25 @@
 #ifndef READ_PLACE_H
 #define READ_PLACE_H
 
+/**
+ * This function is used to read a placement file.
```

Comment:
so that we can skip placement, check netlist file, grid dimensions, if verify flag is set must give error